configure

```diff
@@ -728,7 +728,7 @@ def configure_library(lib, output):
     # libpath needs to be provided ahead libraries
     if pkg_libpath:
       output['libraries'] += (
-          filter(None, map(str.strip, pkg_cflags.split('-L'))))
+          map('-L{0}'.format, filter(None, map(str.strip, pkg_libpath.split('-L')))))
```

Comment:
Whitespace isn't safe -- `/foo/my files/`. Output from pkg_config should avoid being parsed if possible.